Harvest.Web/Views/Shared/_Layout.cshtml

```diff
@@ -112,6 +112,12 @@
             All rights reserved.</p>
     </footer>
 
+
+    <script>
+        var Harvest = { user: {} };
+        Harvest.user.roles = @Html.Raw(await Context.GetUserRoles());
+    </script>
+
```

Comment:
Do roles really need to be pushed in this way? Why not just have a UserRolesPromise initialize on the app page and pass it into AppContextProvider?

Reply:
Could be done either way.  This just seemed easier to me because you have one less http request and one less promise loading state.

Reply:
Hmm. This does eliminate an extra round trip.

Reply:
Dumb question. Could they just edit this? Or we will only use this to show and hide stuff and it will get reloaded each time in that case.

Reply:
@jSylvestre they can't really edit that, but they can easily edit the javascript that reads it.  But yes, this is only to show/hide, we won't be relying on any of this for actual security.